src/Stryker.Core/Stryker.Core/ToolHelpers/VsTestHelper.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Stryker.Core.Options;
+using Stryker.Core.Options.Values;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Stryker.Core.ExecutableFinders
+{
+    public class VsTestHelper
+    {
+        private readonly StrykerOptions _options;
+        private readonly IFileSystem _fileSystem;
+
+        public VsTestHelper(StrykerOptions options, IFileSystem fileSystem = null)
+        {
+            _options = options;
+            _fileSystem = fileSystem ?? new FileSystem();
+        }
+
+        public Dictionary<DotnetFramework, string> GetVsTestToolPaths()
+        {
+            Dictionary<DotnetFramework, string> vsTestPaths = new Dictionary<DotnetFramework, string>();
+            string versionString = "16.0.0-preview-20181205-02";
```

Comment:
Can we get rid of the hard coded version number?

Reply:
I have not been able to come up with a better way for now..